projects/vdk-core/tests/vdk/internal/builtin_plugins/ingestion/test_ingester_base.py

```diff
@@ -14,7 +15,7 @@
 
 shared_test_values = {
     "test_payload1": {"key1": "val1", "key2": "val2", "key3": "val3"},
-    "test_payload2": {"key1": 1, "key2": 2, "key3": 3},
+    "test_payload2": {"key1": Decimal(1), "key2": 2, "key3": 3},
     "test_expected_payload1": [{"key1": "val1", "key2": "val2", "key3": "val3"}],
     "test_expected_payload2": [{"key1": 1, "key2": 2, "key3": 3}],
```

Comment:
That seems wrong. SHouldn't key1 be "Decimal" as well ? We are changing the payload ?

Reply:
amended